From cc96bcbde6556cb412944ddc40f7b6a89fe4ae27 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Fri, 23 Nov 2018 14:22:36 +0800
Subject: [PATCH] linux: fix build when using musl

Alternative definition for CONCAT which is not defined by musl.

Upstream-Status: Pending

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 media_driver/linux/common/os/mos_defs_specific.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/media_driver/linux/common/os/mos_defs_specific.h b/media_driver/linux/common/os/mos_defs_specific.h
index 29a8d566..d8244207 100644
--- a/media_driver/linux/common/os/mos_defs_specific.h
+++ b/media_driver/linux/common/os/mos_defs_specific.h
@@ -81,7 +81,8 @@ typedef struct _TP_CALLBACK_INSTANCE TP_CALLBACK_INSTANCE, *PTP_CALLBACK_INSTANC
 /* compile-time ASSERT */
 
 #ifndef C_ASSERT
-    #define __UNIQUENAME( a1, a2 )  __CONCAT( a1, a2 )
+    #define __CONCATING( a1, a2 )   a1 ## a2
+    #define __UNIQUENAME( a1, a2 )  __CONCATING( a1, a2 )
     #define UNIQUENAME( __text )    __UNIQUENAME( __text, __COUNTER__ )
     #define C_ASSERT(e) typedef char UNIQUENAME(STATIC_ASSERT_)[(e)?1:-1]
 #endif
